src/garage/torch/algos/vpg.py

```diff
@@ -447,13 +444,13 @@ def _compute_objective(self, advantages, obs, actions, rewards):
 
         return log_likelihoods * advantages
 
-    def _process_samples(self, paths):
+    def _process_samples(self, eps):
```

Comment:
is there any way to eliminate this method? that's kind of the idea behind EpisodeBatch

Reply:
The main thing this method does is pad out paths so that we can compute advantages in a differentiable way. I'm not sure that's so important, but it's why `EpisodeBatch` alone does not suffice.